src/components/stepper/stepper.jsx

```diff
@@ -51,7 +54,7 @@ module.exports = React.createClass({
           <Button handleClick={this.decrement} type="button" disabled={!this.canDecrement()}>{this.props.decrementDisplayString}</Button>
         </span>
 
-        <Input type="text" key={this.props.value} readOnly={true}>{this.props.value.toString()}</Input>
+        <Input type="text" id={this.props.id} key={this.props.value} readOnly={true} label={this.props.label}>{this.props.value.toString()}</Input>
```

Comment:
This just passes the label text through to the input component which will then create the label and use the text passed through. I was surprised it was this way around but it solved how i needed it to work so didn't want to mess with the main Input component